website/docs/docs/community-adapters.md

```diff
@@ -7,7 +7,8 @@ Community adapters are adapter plugins contributed and maintained by members of
 
 | Data platforms (click to view setup guide) |||
 | ------------------------------------------ | -------------------------------- | ------------------------------------- |
-| [Clickhouse](/docs/core/connect-data-platform/clickhouse-setup) | [Databend Cloud](/docs/core/connect-data-platform/databend-setup) | [Doris & SelectDB](/docs/core/connect-data-platform/doris-setup)  |
+| [Clickhouse](/docs/core/connect-data-platform/clickhouse-setup) | [CrateDB](/docs/core/connect-data-platform/cratedb-setup)
+| [Databend Cloud](/docs/core/connect-data-platform/databend-setup) | [Doris & SelectDB](/docs/core/connect-data-platform/doris-setup)  |
```

Comment:
> you're missing the link in the Community Adapter section

Thank you. Fixed.

Reply:
> unless you want to be an trusted adapter, then you would want to be linked in the trusted adapter section

I guess that would not be reasonable for a 0.0.2 version. Maybe @kneth would like to follow up with you on that later. What would it take to become a trusted adapter?

Reply:
Right, it's [on the docs](https://docs-getdbt-com-git-fork-crate-workbench-cratedb-dbt-labs.vercel.app/docs/trusted-adapters). This sounds very good.

Reply:
I guess [What it means to be trusted](https://docs-getdbt-com-git-fork-crate-workbench-cratedb-dbt-labs.vercel.app/guides/adapter-creation#what-it-means-to-be-trusted) includes all the important details?

I've created a ticket about it, thanks!

- https://github.com/crate/dbt-cratedb2/issues/9